util/util.go

```diff
@@ -150,3 +139,40 @@ func NewEvent(typed string, expected string, i int) Event {
 		I:        i,
 	}
 }
+
+// Formatting helpers
+func ColumnString(col string, value any) string {
```

Comment:
Close to what I want to remove from the DB column. This function's a bit of a hassle because I need to verify that the value I'm passing in matches what I want based on the column. Is this something I want, or do I want to convert all ints to floats, and then round, based on the output?

Reply:
Keep it as it for now. Don't need to worry about it.